src/popup/components/InfoBox.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div
+    class="info-box"
+    :class="[type]"
+  >
+    <TemplateRenderer :str="content" />
```

Comment:
Why the `TemplateRenderer` is used here? Maybe having just `v-html` would be OK for us?